pkg/sql/tests/rsg_test.go

```diff
@@ -543,6 +543,7 @@ var ignoredErrorPatterns = []string{
 	// TODO(mjibson): fix these
 	"column .* must appear in the GROUP BY clause or be used in an aggregate function",
 	"aggregate functions are not allowed in ON",
+	"ordered-set aggregations must have a WITHIN GROUP clause containing one ORDER BY column",
```

Comment:
huh, i thought this list was only supposed to include AssertionErrors, but in the code it looks like this is actually a pgerror.